wordpress-mobile/WordPress-Android
# Add debug cookies feature – Part 2

This PR adds a "Debug cookies" option to the Debug Settings screen. It basically allows us to add and remove any cookie value to be used on our app requests for testing purposes:


https://user-images.githubusercontent.com/830056/128933846-cdd386ae-d049-41f7-b5d0-3133e392a06c.mp4




### To test

Notes: 
- You'll need a tool like Charles so you can verify that the cookies are being added to the requests.

1. Open the app.
1. Tap the Me button.
1. On the Me screen, go to App Settings > Debug settings > Debug cookies.
1. On the Debug Cookies screen, add any kind of cookie name and cookie value combination (ex: `my_cookie_name` and `my_cookie_value`).
1. Tap "Set cookie".
1. Go back to My Site screen.
1. On Charles, verify that new API requests have the new cookie added to them:

<img width="769" src="https://user-images.githubusercontent.com/830056/128933088-0beb32ad-a219-4a4c-8e82-13951476f8fb.png">

## Regression Notes
1. Potential unintended areas of impact
None that I can think of.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and existing unit tests.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
